deployments/main.go

```diff
@@ -34,12 +33,8 @@ func main() {
 			dp.GetRelayService(namespace),
 			dp.GetRelayDeployment(namespace),
 			dp.GenerateDaemonSet(strings.ToLower(env), namespace),
-			dp.GetPolicyManagerService(namespace),
```

Comment:
no controller? only CRDs?

Reply:
Yes as the controller will require certificates pair to operate and we cannot store  those on git.

However we can update the helm chart to generate them at the during the install process.